integration/nwo/orderer_template.go

```diff
@@ -120,4 +120,5 @@ Metrics:
 ChannelParticipation:
   Enabled: {{ .ChannelParticipationEnabled }}
   MaxRequestBodySize: 1 MB
+  FileRepoDir: {{ .ChannelParticipationFileRepoDir }}
```

Comment:
All orderers share the same location? Seems like a bad idea. Can you explain?

Reply:
Done. Fixed by moving the file repo directory under the configured file ledger location, which is different per orderer node.